services/QuillLMS/client/app/bundles/Diagnostic/components/questions/questionForm.jsx

```diff
@@ -16,23 +16,32 @@ export default class extends React.Component {
   };
 
   submit = () => {
+    const { question, submit, conceptUID } = this.props;
+    const { prompt, instructions, flag, cuesLabel, optimalResponseText, concept } = this.state;
+    const { focusPoints, incorrectSequences, modelConceptUID } = question
     const questionObj = {
-      conceptUID: this.props.question.conceptUID,
-      focusPoints: this.props.question.focusPoints,
-      incorrectSequences: this.props.question.incorrectSequences,
-      modelConceptUID: this.props.question.modelConceptUID,
-      prompt: this.state.prompt,
+      conceptUID: conceptUID,
+      focusPoints: focusPoints,
+      incorrectSequences: incorrectSequences,
+      modelConceptUID: modelConceptUID,
+      prompt: prompt,
       cues: this.refs.cues.value.split(','),
-      instructions: this.state.instructions,
-      flag: this.state.flag,
-      cuesLabel: this.state.cuesLabel
+      instructions: instructions,
+      flag: flag,
+      cuesLabel: cuesLabel
```

Comment:
I think all of the changed values here could now just be `conceptUID, focusPoints, incorrectSequences...` since the values are always the same as the keys.

Reply:
true! I'll update